.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2019-12-16 11:00:09 -0600 using RuboCop version 0.74.0.
+# on 2020-04-01 08:40:11 -0700 using RuboCop version 0.80.0.
```

Comment:
This isn't the version in the Gemfile.lock, is it?

Reply:
That's odd, we pin 0.74.0, why would running it locally use 0.80.0?

Reply:
Updating to 0.80.x

Reply:
you probably ran `rubocop` rather than `bundle exec rubocop`

Reply:
Oh yup. Need to get into that habit.

Reply:
Incidentally, this is one of the reasons why I still use `rvm` with per-project gemsets; I almost never have to use `bundle exec` because it does the right thing.